inference-engine/tests_deprecated/functional/shared_tests/single_layer_tests/ti_tests.hpp

```diff
@@ -177,7 +177,8 @@ class TITestBase: public PlgTest<ti_test_params> {
 
 using TITest  = TITestBase;
 
-TEST_P(TITest, TestsWitUnusedOut) { RunTITest(); }
+// disabled due to transition to ngraph transformations
+TEST_P(TITest, DISABLED_TestsWitUnusedOut) { RunTITest(); }
```

Comment:
Am I right that we don't have a plan to enable back these tests?
If yes, should we remove these tests?

Reply:
I will review the functionality of these tests and try to move it in the current test system

Reply:
Do we have a ticket for this?